src/core/public/overlays/overlay_service.ts

```diff
@@ -77,6 +77,7 @@ export interface OverlayStart {
   openModal: (
     modalChildren: React.ReactNode,
     modalProps?: {
+      className?: string;
```

Comment:
necessary, to add className to the modal, for a fixed height, preventing resizing when changing tabs